packages/key-did-resolver/src/index.ts

```diff
@@ -0,0 +1,29 @@
+import type { ParsedDID, DIDResolver, DIDDocument } from 'did-resolver'
+
+interface ResolverRegistry {
+    [index: string]: DIDResolver;
+}
+
+import * as secp256k1 from './secp256k1/secp256k1-driver'
+
+// supported drivers
+const prefixToDriverMap: any = {
+    zQ3s: secp256k1,
```

Comment:
We should not assume this is encoded with base58btc.

Reply:
Let's check which multibase first, then use `varint` package to parse the first varint to see which pubkey type we have.